microshift_storage/persistent_storage_microshift/microshift-storage-plugin-overview.adoc

```diff
@@ -6,5 +6,142 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
+{product-title} enables dynamic storage provisioning that is ready for immediate use with the logical volume manager storage (LVMS) Container Storage Interface (CSI) provider. The LVMS plugin is the Red Hat downstream version of TopoLVM, a CSI plug-in for managing LVM volumes for Kubernetes. 
 
+LVMS provisions new logical volume management (LVM) logical volumes (LVs) for container workloads with appropriately configured PersistentVolumeClaims (PVC). Each PVC references a StorageClass that represents an LVM Volume Group (VG) on the host node.  LVs are only provisioned for scheduled pods.
```

Comment:
```suggestion
LVMS provisions new logical volume management (LVM) logical volumes (LVs) for container workloads with appropriately configured persistent volume claims (PVC). Each PVC references a storage class that represents an LVM Volume Group (VG) on the host node.  LVs are only provisioned for scheduled pods.
```
See the glossary on [persistent volume claims](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#storage-class) and [storage class](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#storage-class)